index.js

```diff
@@ -7,6 +7,19 @@ module.exports.parse = async ({ message, defaultSchemaFormat }) => {
   message['x-parser-original-payload'] = message.payload;
   message.payload = transformed;
   delete message.schemaFormat;
+
+  async function handleKafkaProtocolKey() {
```

Comment:
One suggestion to add comment in code :)

```suggestion

  // remove that function when https://github.com/asyncapi/spec/issues/622 will be introduced in AsyncAPI spec
  async function handleKafkaProtocolKey() {
```

Reply:
✅